DependencyInjection/Configuration.php

```diff
@@ -30,7 +27,7 @@ public function getConfigTreeBuilder()
         $rootNode
             ->children()
                 ->booleanNode('autoinclude')->defaultTrue()->end()
-                ->scalarNode('base_path')->defaultValue('bundles/norzechowiczaceeditor/ace')->end()
+                ->scalarNode('base_path')->defaultValue('vendor/ace')->end()
```

Comment:
> The directory vendoris not created by composer, it it's in the web folder.

🤦‍♂️ sorry my bad, I was thinking that you are pointing users into `vendor` folder that belongs to composer. Yea, this way a lot nicer and cleaner than it was before, could you just please use `web/vendor/ace` as a default instead of `vendor/ace`?